src/model/race.ts

```diff
@@ -0,0 +1,122 @@
+enum Attributes {
+    STRENGTH,
+    DEXTERITY,
+    INTELLIGENCE,
+    WISDOM,
+    CONSTITUTION,
+    CHARISMA
+};
+
+enum AbilityRefresh {
+    ATWILL,
+    BATTLE,
+    DAILY
+};
+enum AbilityType {
+    STANDARD,
+    MOVE,
+    QUICK,
+    INTERRUPT,
+    FREE
+};
+enum AbilityTrigger {
+    HIT,
+    DAMAGETAKEN,
+    MISS,
+};
+enum ArmorTypes {
+    NONE,
+    LIGHT,
+    HEAVY,
+    SHIELD
+};
+enum MeleeWeapons {
+    ONEHSMALL,
+    ONEHLIGHT,
+    ONEHHEAVY,
+    TWOHSMALL,
+    TWOHLIGHT,
+    TWOHHEAVY
+};
+enum RangedWeapons {
+    THROWNSMALL,
+    THROWNLIGHT,
+    THROWNHEAVY,
+    XBOWSMALL,
+    XBOWLIGHT,
+    XBOWHEAVY,
+    BOWLIGHT,
+    BOWHEAVY
+};
+enum Tiers {
+    ADVENTURER,
+    CHAMPION,
+    EPIC
+}
+export interface IRace {
+    bonusstat1: Attributes,
+    bonusstat2: Attributes,
+    abilityrefresh: AbilityRefresh,
+    abilitytype: AbilityType,
+    abilitytrigger: AbilityTrigger,
+    abilitytext: string
+
+};
+export interface IBackground {
+    backgroundtitle : string,
+    backgroundmod : number
+}
+export interface IFeats {
+    tier: Tiers,
+    prereq: string,
```

Comment:
Not sure a good way to enforce prereqs of feats.  Maybe just pass that on to the user and they can do stupid things like pick a feat for a spell they don't have